iocore/net/SSLUtils.cc

```diff
@@ -308,8 +308,10 @@ set_context_cert(SSL *ssl)
   if (ctx != NULL) {
     SSL_set_SSL_CTX(ssl, ctx);
 #if HAVE_OPENSSL_SESSION_TICKETS
-    // Reset the ticket callback if needed
-    SSL_CTX_set_tlsext_ticket_key_cb(ctx, ssl_callback_session_ticket);
+    if (SSL_CTX_get_options(ctx) & SSL_OP_NO_TICKET) {
```

Comment:
I'm not sure this is correct:

SSL_OP_NO_TICKET
Normally clients and servers will, where possible, transparently make use of RFC4507bis tickets for stateless session resumption.

If this option is set this functionality is disabled and tickets will not be used by clients or servers.